wordpress/wp-content/plugins/memberful-wp/src/widgets.php

```diff
@@ -124,7 +124,7 @@ function memberful_wp_format_widget_links( $links ) {
     $links[$key] = '<a href="'.$link['href'].'" class="'.$link['class'].'">'.$link['text'].'</a>';
   }
 
-  return implode( ' | ', $links );
+  return implode( ' <span class="memberful-links-separator">|</span> ', $links );
```

Comment:
Yes, we had a customer who wanted to hide the separator with CSS because they were styling the widget themselves. This class makes that possible.